packages/ripple-binary-codec/src/types/currency.ts

```diff
@@ -87,12 +87,13 @@ class Currency extends Hash160 {
     super(byteBuf ?? Currency.XRP.bytes)
     const code = this.bytes.slice(12, 15)
```

Comment:
This is now only used in the `else if` case - it can probably just be computed in that section